app/policies/application_policy.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+# Base class for application policies
+class ApplicationPolicy < ActionPolicy::Base
+  # Configure additional authorization contexts here
+  # (`user` is added by default).
+  #
+  #   authorize :account, optional: true
+  #
+  # Read more about authorization context: https://actionpolicy.evilmartians.io/#/authorization_context
+
+  # Define shared methods useful for most policies.
+  # For example:
+  #
+  #  def owner?
+  #    record.user_id == user.id
+  #  end
+  #
+
+  def can_modify?(obj)
+    Member.can_modify?(user, obj)
+  end
+
+  def can_view?(obj)
+    Member.can_view?(user, obj)
+  end
+
+  def can_destroy?(obj)
+    Member.can_destroy?(user, obj)
+  end
+
+  def can_transfer?(obj)
+    Member.namespace_owners_include_user?(user, obj)
+  end
+
+  scope_for :relation, :manageable do |relation|
```

Comment:
Can this be moved to a `namespace_policy.rb` that the namespaces base off of.

Reply:
Moved in 4bfd3d7622524a5c14a274b634ea233e0f4387e9